rust/processor/src/db/postgres/migrations/2024-08-25-070629_add_latest_states_index/up.sql

```diff
@@ -0,0 +1,5 @@
+-- Your SQL goes here
+
+CREATE INDEX latest_states_idx
+ON market_latest_state_event (bump_time DESC)
+INCLUDE (market_id);
```

Comment:
@alnoki @CRBl69 Let's discuss on Monday's call potentially consolidating all `up.sql` and `down.sql` changes before our initial release in cases where it might make sense.

e.g., the `pro_rata_...` field name changes mean the original field names are needlessly misleading. I accidentally even copied the field names over incorrectly because of this. It might be worthwhile to consolidate everything to a single migration as long as we haven't relesaed/deployed anything yet